jsonschema-hyperschema.xml

```diff
@@ -854,6 +855,87 @@
                     Note that these relationship values are case-insensitive, consistent with their
                     use in HTML and the <xref target="RFC5988">HTTP Link header</xref>.
                 </t>
+                <section title="&quot;collection&quot; and &quot;item&quot; links">
+                    <t>
+                        <xref target="RFC6573">RFC 6573</xref> defines and registers
+                        the "item" and "collection" link relations.  A well-known design pattern
+                        in hypermedia is to use a collection resource to create a member of the
+                        collection and give it a server-assigned URI.  When using HTTP, or a protocol
+                        such as CoAP that is explicitly analogous to HTTP, this is done by POST-ing
+                        a representation of the individual resource to be created to the collection
+                        resource.
+                    </t>
+                    <t>
+                        Resources that are the target of a "collection" link using HTTP or an analogous
+                        protocol in JSON Hyper-Schema MUST NOT assign semantics other than resource
+                        creation to POST (or the analogous method in non-HTTP protocols).
+                    </t>
+                    <t>
+                        The <xref target="submissionSchema">"submissionSchema"</xref> field for the
+                        link SHOULD be identical to the schema of the representations of the
```

Comment:
What do you mean by "identical" here?
For instance, if I'd like to add constraints on the `submissionSchema` (like a `required` or the `"readOnly": true` from the example below, assuming it's not already on `/thing`'s schema), is this allowed?

Reply:
This wasn't supposed to say "identical", actually.  I just kept rewriting it, and then changing ti back, and apparently forgot I still needed to work on it.  At various times it was "analogous to" or "compatible with".

What I want to convey is that the representation used to create a resource here should be more-or-less the same as the one used in "targetSchema" for the individual resource's "self" link.  But It can, of course, differ in things like the server-generated "id" being absent during creation but required in "targetSchema".  I actually handle that by just marking it read-only and not required in the schema but documenting that it's always present when retrieved from the server.

I'm not sure how to explain exactly what should and shouldn't be allowed here.  I don't even want there to be some hard and fast rule, and it definitely needs to be a SHOULD or RECOMMENDED instead of a MUST.

Any suggestions welcome.

Reply:
In the postamble of the example, I used "matching", but didn't explain what that meant either.

Reply:
"compatible with" sounds good to me; what you meant is clear enough I think (I just ticked on "identical", but now that's fine) and follows common sense so that it probably does not need more explanation.